__tests__/organisationClient.ts

```diff
@@ -150,13 +270,130 @@ describe('organisationClient', () => {
               }
             ],
             "name": "Kassulke Inc",
-            "organisationIdentifier": "24-3701590"
+            "organisationIdentifier": "23-3333333"
           }
         ]);
       })
     });
 
-    test('it should return an empty list if no organisations include organiation name', () => {
+    test('it should return a list of organisations when matching organisations are found in Ascending alphabetical order', () => {
```

Comment:
I can understand what somebody means by "alphabetical order", but I'm not sure what "Ascending alphabetical order" means.

Does it mean that A comes before B? :) Is this not just an "alphabetical order".

Reply:
Yeah this comes down to nuances of language so i thought it would be best to specify.
You can sort in two ways Alphabetically. Ascending and Decending.
Ascending is "Apple. Banana,Pear"
Decending is "Pear, Banana, Apple"
Most people understand "alphabetical order" but as we have to deal with the chance that a company is called "1 Solution Inc". Its important to know that the sorting is done in Ascending order from an ASCII table.
So from the title saying (Ascending) we can infer to expect this as the order:
 ` # $ % & ' () * + , -./ 0 1 2 3 4 5 6 7 8 9 :;< = >? @A B C D E F G H I J K L M N O P Q R S T U V W X Y Z[\] ^ _ ` {|} ~`
So we do not expect numbers to appear after letters

Reply:
ok